src/main/java/configuration/Config.java

```diff
@@ -81,6 +82,7 @@ public static void init() throws Exception {
         if (USE_SASL_AUTH) {
             SASL_USERNAME = getString(dotenv, "SASL_USERNAME");
             SASL_PASSWORD = getStringValueOrFromFile(dotenv, "SASL_PASSWORD");
+            SASL_MECHANISM = getOptionalString(dotenv, "SASL_MECHANISM", "plain");
```

Comment:
Should be uppercase? It was before

Reply:
changed to upper, and anyway I'm doing toUpperCase in `KafkaClientFactory.java` to avoid user mistakes